app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,14 @@
+<article class="recipe"> 
```

Comment:
Good job 👍🏽, but it would be nicer if you add a link where the logged_user can create a new recipe